### ISSUE TITLE
Step by Step A11y issue - Keyboard navigation fail

### ISSUE THREAD

**Issue by artuoma** [cagov/Alpha#243]
**Describe the bug**
Any page using the step by step content type is not controllable by keyboard. 

**To Reproduce**
1. Go to 'https://alpha.ca.gov/services/hire-licensed-contractor-home-improvements/'
2. Start Tabbing through the page using the keyboard
3. Once you get to the step by step module
4. Cannot open or close the module using the keyboard

**Expected behavior**
Complete control of the module using the keyboard's TAB and Enter keys. See example:
- https://www.gov.uk/get-tax-free-childcare

**Issue Resources**
- https://www.w3.org/WAI/GL/wiki/Making_actions_keyboard_accessible_by_using_keyboard_event_handlers_with_WAI-ARIA_controls

**Comment by artuoma** [comment]
@aaronhans FYI

**Comment by aaronhans** [comment]
Made the top level elements into buttons and made the interior hide and reappear with display:none with delay after animation completes, storing initial height as an attribute on the HTML element object in the DOM (not a visible attribute in the source) https://github.com/cagov/Alpha/pull/261

**Comment by abquirarte** [comment]
✅ Done 😄
